src/pages/resources/Resources.css

```diff
@@ -1,3 +1,44 @@
-.links {
-  display: block;
-}
\ No newline at end of file
+.resources_wrapper {
+  height: 100%;
+}
+.top {
+  text-align: center;
+  padding: 4rem;
+}
+.bottom {
+  width: 80%;
+  margin: 0 auto;
+}
+.resourcesList {
```

Comment:
I don't have a strong opinion about this since we are not following any style guide here (yet), but usually there's a blank line between CSS rule blocks. No need to change anything, just FYI